package.json

```diff
@@ -102,5 +102,6 @@
   },
   "resolutions": {
     "graceful-fs": "^4.2.4"
-  }
+  },
+  "packageManager": "yarn@4.1.1"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** I'm a bit confused about this. We are adding `.yarn/releases/yarn-classic.cjs` file so that newer version of yarn won't try to update to v4, then we set v4 as default package manager. Am I missing something?‏

Reply:
`packageManager` tells `corepack` to load `yarn@4.1.1` that then loads `yarn classic` according to the `yarnrc.yml` configuration

the commit files was generated by running the command `yarn set version classic`, and according to the [yarn set version documentation](https://yarnpkg.com/cli/set/version)

> This command will set a specific release of Yarn to be used by Corepack: https://nodejs.org/api/corepack.html.

> By default it only will set the [packageManager](https://yarnpkg.com/configuration/manifest#packageManager)  field at the root of your project, but if the referenced release cannot be represented this way, if you already have [yarnPath](https://yarnpkg.com/configuration/yarnrc#yarnPath) configured, or if you set the --yarn-path command line flag, then the release will also be downloaded from the Yarn GitHub repository, stored inside your project, and referenced via the [yarnPath](https://yarnpkg.com/configuration/yarnrc#yarnPath) settings from your project .yarnrc.yml file.

I think that `classic` is not a valid `packageManager` value for `corepack` - maybe tomorrow I can try to set `1.22`